tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -189,9 +189,14 @@ def tearDown(self):
         shutil.rmtree(ARTIFACT_URI)
 
     def _experiment_factory(self, names):
-        if type(names) is list:
-            return [self.store.create_experiment(name=name) for name in names]
+        if isinstance(names, (list, tuple)):
+            ids = []
+            for name in names:
+                time.sleep(0.001)
```

Comment:
Can we add some inline comments explaining why we sleep for 1 millisecond?

Reply:
Sure!